acvm/src/pwg/hash.rs

```diff
@@ -41,6 +42,23 @@ pub fn sha256(
     Ok(OpcodeResolution::Solved)
 }
 
+pub fn keccak(
```

Comment:
nit but maybe rename to `keccak256` to be specific? (Granted we don't do this for Blake)

Reply:
Yep done!